controllers/managedocs_controller_test.go

```diff
@@ -777,6 +785,17 @@ var _ = Describe("ManagedOCS controller", func() {
 					Expect(secret.Data["key"]).Should(Equal([]byte("-----BEGIN PUBLIC KEY-----\ntest-validation-key\n-----END PUBLIC KEY-----")))
 				})
 			})
+			When("it is a provider deployment", func() {
+				It("should have rook-config-override configmap settings", func() {
+					if testReconciler.DeploymentType != providerDeploymentType {
+						Skip(fmt.Sprintf("Skipping the test as it is not required by %v", testReconciler.DeploymentType))
+					}
+					configMap := rookOverrideConfigMapTemplate.DeepCopy()
+					utils.WaitForResource(k8sClient, ctx, configMap, timeout, interval)
+					fmt.Println(configMap.Data["config"])
```

Comment:
Why are you using `fmt.Println` inside a test?

Reply:
Should'nt be there. Have removed it.